File: russfest/go/russfestimpl/funcs.go
package russfestimpl

import (
	"github.com/iotaledger/wasp/packages/wasmvm/wasmlib/go/wasmlib"
)

func funcAcceptShop(ctx wasmlib.ScFuncContext, f *AcceptShopContext) {
}

func funcAddMusician(ctx wasmlib.ScFuncContext, f *AddMusicianContext) {
}

func funcBuyMerch(ctx wasmlib.ScFuncContext, f *BuyMerchContext) {
}

func funcCallCheckProduct(ctx wasmlib.ScFuncContext, f *CallCheckProductContext) {
}

func funcCallCreateNewProduct(ctx wasmlib.ScFuncContext, f *CallCreateNewProductContext) {
}

func funcCallPayStore(ctx wasmlib.ScFuncContext, f *CallPayStoreContext) {
}

func funcCallPingShop(ctx wasmlib.ScFuncContext, f *CallPingShopContext) {
}

func funcCallRecieveProducts(ctx wasmlib.ScFuncContext, f *CallRecieveProductsContext) {
}

func funcCallSendProduct(ctx wasmlib.ScFuncContext, f *CallSendProductContext) {
}

func funcCancelShopRequest(ctx wasmlib.ScFuncContext, f *CancelShopRequestContext) {
}

func funcDenyShop(ctx wasmlib.ScFuncContext, f *DenyShopContext) {
}

func funcInit(ctx wasmlib.ScFuncContext, f *InitContext) {
	if f.Params.Owner().Exists() {
		f.State.Owner().SetValue(f.Params.Owner().Value())
		return
	}
	f.State.Owner().SetValue(ctx.RequestSender())
}

func funcRequestShopLicence(ctx wasmlib.ScFuncContext, f *RequestShopLicenceContext) {
}

func funcSetOwner(ctx wasmlib.ScFuncContext, f *SetOwnerContext) {
	f.State.Owner().SetValue(f.Params.Owner().Value())
}

func funcUpdateDeniedShopRequest(ctx wasmlib.ScFuncContext, f *UpdateDeniedShopRequestContext) {
}

func viewCallGetShopStatistics(ctx wasmlib.ScViewContext, f *CallGetShopStatisticsContext) {
}

func viewGetAgendID(ctx wasmlib.ScViewContext, f *GetAgendIDContext) {
}

func viewGetAllOpenShopRequests(ctx wasmlib.ScViewContext, f *GetAllOpenShopRequestsContext) {
}

func viewGetAllProducts(ctx wasmlib.ScViewContext, f *GetAllProductsContext) {
}

func viewGetAllRegisteredShops(ctx wasmlib.ScViewContext, f *GetAllRegisteredShopsContext) {
}

func viewGetDeniedShopRequests(ctx wasmlib.ScViewContext, f *GetDeniedShopRequestsContext) {
}

func viewGetErrorMessagesView(ctx wasmlib.ScViewContext, f *GetErrorMessagesViewContext) {
}

func viewGetFestivalEarnings(ctx wasmlib.ScViewContext, f *GetFestivalEarningsContext) {
}

func viewGetMusicians(ctx wasmlib.ScViewContext, f *GetMusiciansContext) {
}

func viewGetMusiciansWithoutShop(ctx wasmlib.ScViewContext, f *GetMusiciansWithoutShopContext) {
}

func viewGetOpenShopRequest(ctx wasmlib.ScViewContext, f *GetOpenShopRequestContext) {
}

func viewGetOwner(ctx wasmlib.ScViewContext, f *GetOwnerContext) {
	f.Results.Owner().SetValue(f.State.Owner().Value())
}

func viewGetRegisteredShopsFromOwner(ctx wasmlib.ScViewContext, f *GetRegisteredShopsFromOwnerContext) {
}

func viewGetShopStatistics(ctx wasmlib.ScViewContext, f *GetShopStatisticsContext) {
}

func viewGetSpecificProducts(ctx wasmlib.ScViewContext, f *GetSpecificProductsContext) {
}

func viewGetTimeslots(ctx wasmlib.ScViewContext, f *GetTimeslotsContext) {
}

func viewTestView(ctx wasmlib.ScViewContext, f *TestViewContext) {
}

func viewTestViewSingle(ctx wasmlib.ScViewContext, f *TestViewSingleContext) {
}

File: russfest/ts/russfestimpl/funcs.ts
import * as wasmlib from 'wasmlib';
import * as sc from '../russfest/index';

export function funcAcceptShop(ctx: wasmlib.ScFuncContext, f: sc.AcceptShopContext): void {
}

export function funcAddMusician(ctx: wasmlib.ScFuncContext, f: sc.AddMusicianContext): void {
}

export function funcBuyMerch(ctx: wasmlib.ScFuncContext, f: sc.BuyMerchContext): void {
}

export function funcCallCheckProduct(ctx: wasmlib.ScFuncContext, f: sc.CallCheckProductContext): void {
}

export function funcCallCreateNewProduct(ctx: wasmlib.ScFuncContext, f: sc.CallCreateNewProductContext): void {
}

export function funcCallPayStore(ctx: wasmlib.ScFuncContext, f: sc.CallPayStoreContext): void {
}

export function funcCallPingShop(ctx: wasmlib.ScFuncContext, f: sc.CallPingShopContext): void {
}

export function funcCallRecieveProducts(ctx: wasmlib.ScFuncContext, f: sc.CallRecieveProductsContext): void {
}

export function funcCallSendProduct(ctx: wasmlib.ScFuncContext, f: sc.CallSendProductContext): void {
}

export function funcCancelShopRequest(ctx: wasmlib.ScFuncContext, f: sc.CancelShopRequestContext): void {
}

export function funcDenyShop(ctx: wasmlib.ScFuncContext, f: sc.DenyShopContext): void {
}

export function funcInit(ctx: wasmlib.ScFuncContext, f: sc.InitContext): void {
    if (f.params.owner().exists()) {
        f.state.owner().setValue(f.params.owner().value());
        return;
    }
    f.state.owner().setValue(ctx.requestSender());
}

export function funcRequestShopLicence(ctx: wasmlib.ScFuncContext, f: sc.RequestShopLicenceContext): void {
}

export function funcSetOwner(ctx: wasmlib.ScFuncContext, f: sc.SetOwnerContext): void {
    f.state.owner().setValue(f.params.owner().value());
}

export function funcUpdateDeniedShopRequest(ctx: wasmlib.ScFuncContext, f: sc.UpdateDeniedShopRequestContext): void {
}

export function viewCallGetShopStatistics(ctx: wasmlib.ScViewContext, f: sc.CallGetShopStatisticsContext): void {
}

export function viewGetAgendID(ctx: wasmlib.ScViewContext, f: sc.GetAgendIDContext): void {
}

export function viewGetAllOpenShopRequests(ctx: wasmlib.ScViewContext, f: sc.GetAllOpenShopRequestsContext): void {
}

export function viewGetAllProducts(ctx: wasmlib.ScViewContext, f: sc.GetAllProductsContext): void {
}

export function viewGetAllRegisteredShops(ctx: wasmlib.ScViewContext, f: sc.GetAllRegisteredShopsContext): void {
}

export function viewGetDeniedShopRequests(ctx: wasmlib.ScViewContext, f: sc.GetDeniedShopRequestsContext): void {
}

export function viewGetErrorMessagesView(ctx: wasmlib.ScViewContext, f: sc.GetErrorMessagesViewContext): void {
}

export function viewGetFestivalEarnings(ctx: wasmlib.ScViewContext, f: sc.GetFestivalEarningsContext): void {
}

export function viewGetMusicians(ctx: wasmlib.ScViewContext, f: sc.GetMusiciansContext): void {
}

export function viewGetMusiciansWithoutShop(ctx: wasmlib.ScViewContext, f: sc.GetMusiciansWithoutShopContext): void {
}

export function viewGetOpenShopRequest(ctx: wasmlib.ScViewContext, f: sc.GetOpenShopRequestContext): void {
}

export function viewGetOwner(ctx: wasmlib.ScViewContext, f: sc.GetOwnerContext): void {
    f.results.owner().setValue(f.state.owner().value());
}

export function viewGetRegisteredShopsFromOwner(ctx: wasmlib.ScViewContext, f: sc.GetRegisteredShopsFromOwnerContext): void {
}

export function viewGetShopStatistics(ctx: wasmlib.ScViewContext, f: sc.GetShopStatisticsContext): void {
}

export function viewGetSpecificProducts(ctx: wasmlib.ScViewContext, f: sc.GetSpecificProductsContext): void {
}

export function viewGetTimeslots(ctx: wasmlib.ScViewContext, f: sc.GetTimeslotsContext): void {
}

export function viewTestView(ctx: wasmlib.ScViewContext, f: sc.TestViewContext): void {
}

export function viewTestViewSingle(ctx: wasmlib.ScViewContext, f: sc.TestViewSingleContext): void {
}
